### ISSUE TITLE
Trouble compiling examples

### ISSUE THREAD

**Issue by ryanmrichard** [m-a-d-n-e-s-s/madness#170]
I am trying to build Madness, but keep getting a compiler error for the example files:

```
/XXX/Madness/src/examples/cc2.cc(52): catastrophic error: cannot open source file "chem/CC2.h"
  #include <chem/CC2.h>

```
The compiling command CMake generates is:
```
cd /XXX/Madness/src/Madness-build/src/examples && /XXX/mpic++   -DMPICH_SKIP_MPICXX=1 -DOMPI_SKIP_MPICXX=1 -std=c++11 -I/XXX/Madness/src/Madness-build/src/examples -I/XXX/Madness/src/Madness/src/examples -I/XXX/Madness/src/Madness/src -I/XXX/Madness/src/Madness-build/src -I/XXX/Madness/src/Madness-build/src/apps/chem -I/XXX/Madness/src/Madness/src/apps/chem -I/XXX/Madness/src/Madness-build/src/madness/mra -I/XXX/Madness/src/Madness/src/madness/mra -I/XXX/Madness/src/Madness-build/src/madness/tensor -I/XXX/Madness/src/Madness/src/madness/tensor -I/XXX/Madness/src/Madness-build/src/madness/misc -I/XXX/Madness/src/Madness/src/madness/misc -I/XXX/tbb/include -I/XXX/MVAPICH2_intel/include -I/XXX/Madness/src/Madness-build/src/madness/world -IXXX/Madness/src/Madness/src/madness/world -I/XXX/Madness/src/Madness-build/src/madness/external/tinyxml -I/XXX/Madness/src/Madness/src/madness/external/tinyxml -I/XXX/Madness/src/Madness-build/src/madness/external/muParser -I/XXX/Madness/src/Madness/src/madness/external/muParser    -o CMakeFiles/cc2.dir/cc2.cc.o -c /XXX/Madness/src/Madness/src/examples/cc2.cc
```
the problem is that there is no include for `/XXX/Madness/src/Madness-build/src/apps`, alternatively you need to remove the `chem/` from inside `cc2.cc`.

Configuration wise, I'm building it as an external CMake project, with no fancy options (the compilers are set to the MPI-wrappers):
```CMake
      PREFIX "${CMAKE_CURRENT_BINARY_DIR}/Madness"
      GIT_REPOSITORY https://github.com/m-a-d-n-e-s-s/madness
      GIT_TAG 2383898aa6701beb1205d86ac2cf424cf8611933
      UPDATE_COMMAND ""
      CMAKE_COMMAND ${CMAKE_COMMAND}
      CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}
              -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
              -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
              -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
              -DBLAS_LIBRARIES=-mkl
              -DENABLE_UNITTESTS=FALSE
      INSTALL_DIR ${CMAKE_CURRENT_BINARY_DIR}
```

**Comment by evaleev** [comment]
The cmake harness is experimental and not fully "supported" yet. Will fix shortly.

**Comment by evaleev** [comment]
addressed in ef14fac0d1b6ca711916edaaa06b5f7876abc1ef , please try.
